checker/tests/lock/JCIPAnnotations.java

```diff
@@ -1,95 +1,264 @@
+// @skip-test
```

Comment:
There isn't. I'll bring this up at tomorrow's meeting, because we may just want to delete this file. I don't foresee us supporting JCIP annotations anytime soon. It's too much work to interpret JCIP's @GuardedBy as our @GuardedBy for variables and our @Holding for methods and it doesn't seem worth the effort.